iina/MPVCommand.swift

```diff
@@ -10,12 +10,8 @@ struct MPVCommand: RawRepresentable {
 
   init?(rawValue: RawValue) { self.rawValue = rawValue }
 
-  /** ignore */
-  static let ignore = MPVCommand("ignore")
   /** seek <target> [<flags>] */
```

Comment:
My rule is that generated files should never be checked into source control. They should always be regenerated as a part of the build. Too often I have encountered generated files where developers modified the generated files instead of correcting the true source or the generator that created the generated source. I have even countered the situation where the true source and the generator have been lost.

IINA is violating this rule. The swift classes `MPVCommand`, `MPVOption` and `MPVProperty` are all generated and must not be modified by developers. These files are generated from mpv documentation by the script `other/parse_doc.rb` and then checked into git. This is done whenever IINA upgrades to a newer version of the mpv library.

Another of my own rules is being violated, which is that all source code generator must start the generated code with a BIG comment that tries to make it clear to developers that this is a generated file that must not be directly modified by developers. Possibly that comment should contain details about the generation. In this case it probably should indicate the mpv version the source was generated from.

Within Xcode these sources aren't even in their own "Generated" group. The [Current structure](https://github.com/iina/iina/blob/develop/CONTRIBUTING.md#current-structure) section of the [Contributing](https://github.com/iina/iina/blob/develop/CONTRIBUTING.md) document does not warn contributors about these generated sources.

I just entered issue #4429 that suggests improvements in this area.

Reply:
@low-batt hey, thanks for your review! I didn't really know that these classes are auto-generated, I agree with your take here. 
Fixed in ce4a8bca1c16f0f7077a75b0fbe75b010cc44c95